source/cloud/cloud-administration/sso-office.md

```diff
@@ -3,36 +3,75 @@ ___
 
 Follow these steps to configure Mattermost to use your Office 365 logon credentials and Azure Active Directory account as a Single Sign-on (SSO) service for team creation, account creation, and sign-in.
 
-**The system must be using SSL for use with Office365 as Microsoft only allows OAuth redirect URIs that are SSL enabled.**
+**Note:** The system must be using SSL for use with Office365 as Microsoft only allows OAuth redirect URIs that are SSL enabled.
```

Comment:
@sbishel Given that SSL is considered older, less secure terminology (deprecated in favour of TLS), does it make sense to update this note to use TLS instead? CC @wiersgallak

Reply:
Have we changed it in other places? SSL is generally used in a more generic sense and is probably ok to leave here.

Reply:
Fair point - we do use the term "SSL" throughout the docs to refer to something more generic. We'll leave this as-is. Thanks!